sys-apps/etckeeper/etckeeper-1.18.6-r1.ebuild

```diff
@@ -55,6 +55,13 @@ src_install(){
 		exeinto /etc/cron.daily
 		newexe debian/cron.daily etckeeper
 	fi
+
+	# remove subdirectories of /var which cause QA warnings
+	# bug #513374
+	rmdir "${ED%/}"/var/cache/etckeeper >/dev/null ||
```

Comment:
Does redirecting the output to /dev/null fix something here? I tried a bunch of stuff, and `rmdir` is usually silent when it works (and I think it should work in this case?)